src/toil_scripts/gatk_processing/gatk_preprocessing.py

```diff
@@ -482,30 +289,32 @@ def realigner_target_creator(job, shared_ids, input_args):
 
     docker_call(work_dir=work_dir, parameters=parameters,
                 tool='quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
-                inputs=['ref.fa','sample.mkdups.bam', 'ref.fa.fai', 'ref.dict',
-                        'sample.mkdups.bam.bai', 'phase.vcf', 'mills.vcf'],
-                outputs={'sample.intervals': None},
-                env={'JAVA_OPTS':'-Xmx%sg' % input_args.memory})
-    shared_ids['sample.intervals'] = job.fileStore.writeGlobalFile(output)
-    job.addChildJobFn(indel_realignment, shared_ids, input_args)
+                inputs=inputs.keys(), outputs=outputs,
+                env={'JAVA_OPTS':'-Xmx%sg' % xmx},
+                mock=config['mock'])
+    return job.fileStore.writeGlobalFile(output)
 
 
-def indel_realignment(job, shared_ids, input_args):
+def gatk_indel_realigner(job, bamFileStoreID, baiFileStoreID, intervalsFileStoreID, config, memory=8):
```

Comment:
This exists in `toil_scripts.tools.preprocessing` already; remove/consolidate differences.